DailyReportWeb/Web.config

```diff
@@ -98,7 +98,7 @@
   </runtime>
   <system.webServer>
     <modules runAllManagedModulesForAllRequests="true">
-      <remove name="WebDAVModule"/>
+      <remove name="WebDAVModule" />
```

Comment:
Sa mearga http verbs pe IIS [put/delete] insa nu stiu cum de a ajuns in commit-ul lui Sebi